sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -48,6 +51,26 @@ export class EducationProgramController {
     }));
   }
 
+  @Get(":programId")
+  async get(
+    @Param("programId") programId: number,
+  ): Promise<EducationProgramDto> {
+    const educationProgram = await this.programService.getLocationPrograms(
+      programId,
+    );
+
+    return {
+      id: educationProgram.id,
+      name: educationProgram.name,
+      description: educationProgram.description,
+      credentialType: educationProgram.credentialTypeToDisplay,
```

Comment:
@abschwenker, for credential type, are we going to display in the UI the "other credential" when "other" is selected?

Reply:
Yes that would be ideal